unit-tests/src/test/scala/scala/scalanative/posix/ResourceTest.scala

```diff
@@ -0,0 +1,182 @@
+package scala.scalanative
+package posix
+package sys
+
+import org.junit.Test
+import org.junit.Assert._
+
+import scalanative.libc.errno
+import scalanative.posix.errno._
+import scalanative.unsafe.{CInt, Ptr, Zone, alloc}
+import scalanative.unsigned._
+
+import resource._, resourceOps._
+import timeOps._
+
+// Design notes:
+//
+//   * The two methods setpriority() & setrlimit() make changes to the
+//     process execution environment. For that reason they are not tested
+//     here.
+//
+//     unit-tests run sequentially in the same process, so a change caused
+//     by either of those methods could affect later tests in ways that are
+//     hard to trace.
```

Comment:
Maybe for now we should move this test to `tests-ext` to lower chance of interference between tests?

Reply:
Wojciech,  thank you for the review.

I can move the tests to wherever you direct, but I think the current location
is the proper place for two reasons:

1) The Test, as is, as explained in the comment does not exercise the
     potentially problematic methods.  It should be as safe as any other
     Test currently in the ensemble.  

     Being in the same place as the other posixlib tests makes it easy to find.   
     Granted that is a concern only after the Test is shown to be and
     is considered safe.

     The two problematic methods _could_ be tested  using a 
      `scripted` test.  The time/benefit ratio of requiring such a test,
      is not advantageous, especially when there has been discussion
      about removing/limiting `scripted` tests.  

2)  From reverse engineering the contents of `tests-ext`, it looks like
     the purpose of that project is to support parts of Scala Native
     outside the core.  `posixlib sys/resource.scala` should be part of
     the core of Scala Native, at least until some OS dependent spinoffs
     are designed and implemented.  

Please advise.